script.js

```diff
@@ -0,0 +1,87 @@
+// reference to markup elements
+const bodyEle = document.querySelector('body');
+const formsection = document.createElement('section');
+bodyEle.appendChild(formsection);
+
+// list of books
+const bookList = [];
+
+// function to update book list
+function updateBookList() {
+  const bookListElem = document.querySelector('.booklist');
+  bookListElem.innerHTML = '';
+
+  for (let i = 0; i < bookList.length; i += 1) {
+    bookListElem.innerHTML += `
+    <ul class = 'booklst'>
+      <li>${bookList[i].title}</li>
+      <li> ${bookList[i].author}</li>
+        <button class ='btnr' type="button" onclick="removeBook(${i})">Remove</button>
+      </li>
+      <hr class='hr'>
+    </ul>
+    `;
+  }
+}
+
+// add initial book list section
+const bookListSection = document.createElement('section');
+bookListSection.classList.add('booklist');
+formsection.appendChild(bookListSection);
+
+// add initial books to book list section
+updateBookList();
+
+// add form section with "Add" button
+const formSection = document.createElement('section');
+formSection.innerHTML = `
+  <form class='booklst'>
+    <input type="text" name="title" id="booktitle" placeholder="Title">
+    <input type="text" name="author" id="bookauth" placeholder="Author">
+    <button type="button"  class='addbtn' onclick="addBook()">Add</button>
+  </form>
+`;
+formsection.appendChild(formSection);
+
+// function to add a new book to the list
+// eslint-disable-next-line no-unused-vars
+function addBook() {
+  const titleInput = document.querySelector('#booktitle');
+  const authorInput = document.querySelector('#bookauth');
+  const newBook = {
+    title: titleInput.value,
+    author: authorInput.value,
+  };
+  bookList.push(newBook);
+  titleInput.value = '';
+  authorInput.value = '';
+  updateBookList();
+}
+
+// function to remove a book from the list
+// eslint-disable-next-line no-unused-vars
+function removeBook(index) {
+  bookList.splice(index, 1);
+  updateBookList();
+}
```

Comment:
- [ ] Clicking the remove button should remove the selected book from both the app in the browser and from localstorage. Currently the book persists in localstorage after deletion. Kindly fix this